### PR TITLE
update dependencies for 2.2.0 release

### PR DESCRIPTION
**Pull request by jyaistMap**
* updating dependencies in the [system requirements notebook](https://github.com/Esri/arcgis-python-api/blob/master/guide/01-getting-started/system-requirements.ipynb) for the 2.2.0 release
  * based on the source code repo _meta.yaml_ file

* Screenshot of local build (dark line is not from website, it's from screenshot):
image
image
image

* resolves ArcGIS/geosaurus#10400

### CODE REVIEW

File: guide/01-getting-started/system-requirements.ipynb [hunk 1]
```diff
@@ -22,44 +22,57 @@
...
    "source": [
     "## Operating System \n",
-    "The ArcGIS API for Python is compatible with 32-bit and 64-bit versions of Windows, macOS, and Linux.\n",
+    "The ArcGIS API for Python 2.2.0 is compatible with 64-bit versions of Windows, macOS, and Linux. Note that, 32-bit versions of Windows and Linux is no longer supported for API versions later than 1.7.1.\n",
     "\n",
     "## Python Version\n",
-    "Python 3.5 or later is required to use the ArcGIS API for Python.\n",
+    "Python 3.9.x to 3.11.x is required to use the ArcGIS API for Python 2.2.0.\n",
+    "\n",
+    "> **Note:** The `arcgis.learn` module is not supported with Python 3.11.x.\n",
```
Comment (jtroe) [inline guide/01-getting-started/system-requirements.ipynb:30]: or 3.10 as of currently.  Maybe rephrase as arcgis.learn is only supported on 3.9?

File: guide/01-getting-started/system-requirements.ipynb [hunk 2]
```diff
@@ -22,44 +22,57 @@
    "metadata": {},
    "source": [
     "## Operating System \n",
-    "The ArcGIS API for Python is compatible with 32-bit and 64-bit versions of Windows, macOS, and Linux.\n",
+    "The ArcGIS API for Python 2.2.0 is compatible with 64-bit versions of Windows, macOS, and Linux. Note that, 32-bit versions of Windows and Linux is no longer supported for API versions later than 1.7.1.\n",
```
Comment (jtroe) [inline guide/01-getting-started/system-requirements.ipynb:25]:
```suggestion
    "The ArcGIS API for Python 2.2.0 is compatible with 64-bit versions of Windows, macOS, and Linux. Note that, 32-bit versions of Windows and Linux are no longer supported for API versions later than 1.7.1.\n",
```
are?

File: guide/01-getting-started/system-requirements.ipynb [hunk 3]
```diff
@@ -22,44 +22,57 @@
...
     "* [matplotlib](https://matplotlib.org/)\n",
+    "* [matploglib-inline](https://pypi.org/project/matplotlib-inline/)\n",
     "* [notebook](https://ipython.org/notebook.html)\n",
-    "* [ipywidgets >=7](https://ipywidgets.readthedocs.io/en/stable/)\n",
+    "* [ipywidgets >=7,<8](https://ipywidgets.readthedocs.io/en/stable/)\n",
     "* [widgetsnbextension >=3](https://pypi.python.org/pypi/widgetsnbextension)\n",
-    "* [keyring>=23.3.*](https://pypi.python.org/pypi/keyring/10.6.0)\n",
-    "* [winkerberos](https://pypi.python.org/pypi/winkerberos/0.7.0) (Windows only)\n",
-    "* [urllib3](https://pypi.python.org/pypi/urllib3)\n",
+    "* [keyring>=23.3.0](https://pypi.python.org/pypi/keyring/10.6.0)\n",
```
Comment (jtroe) [inline guide/01-getting-started/system-requirements.ipynb:50]: link still pointing to old version

File: guide/01-getting-started/system-requirements.ipynb (hunk 1 above)
Comment (jyaistMap) [reply guide/01-getting-started/system-requirements.ipynb]: Done!

File: guide/01-getting-started/system-requirements.ipynb (hunk 2 above)
Comment (jyaistMap) [reply guide/01-getting-started/system-requirements.ipynb]: oops :) Changed it.

File: guide/01-getting-started/system-requirements.ipynb (hunk 3 above)
Comment (jyaistMap) [reply guide/01-getting-started/system-requirements.ipynb]: Thanks! Updated the link.

File: guide/01-getting-started/system-requirements.ipynb [hunk 4]
```diff
@@ -22,44 +22,57 @@
    "metadata": {},
    "source": [
     "## Operating System \n",
-    "The ArcGIS API for Python is compatible with 32-bit and 64-bit versions of Windows, macOS, and Linux.\n",
+    "The ArcGIS API for Python 2.2.0 is compatible with 64-bit versions of Windows, macOS, and Linux. Note that, 32-bit versions of Windows and Linux are no longer supported for API versions later than 1.7.1.\n",
     "\n",
     "## Python Version\n",
-    "Python 3.5 or later is required to use the ArcGIS API for Python.\n",
+    "Python 3.9.x to 3.10.x is required to use the ArcGIS API for Python 2.2.0.\n",
```
Comment (jtroe) [inline guide/01-getting-started/system-requirements.ipynb:28]:
```suggestion
    "Python 3.9.x to 3.11.x is required to use the ArcGIS API for Python 2.2.0.\n",
```
As far as I know we're still releasing the API for 3.11 even though we're not supporting Learn

**Review by jtroe — APPROVED** [review]
Resolve the py 3.11 comment and this should be good.  Thanks.
